docs/releases/hero-samples.md

```diff
@@ -0,0 +1,107 @@
+
+While the hero samples are based on the samples in this repository, there are some differences:
+
+- The samples in this repository use the tooling required to work in a monorepository (`rush` etc.) whereas the hero samples use vanilla `npm`.
+- The samples in this repository are intended to aid development of the library. Thus, they often use `beta` features of `@azure/communication-react` that are still under active development. In contrast, the hero samples only use `stable` features.
+  - As a corollary, the samples here use [conditional compilation](../references/beta-only-features.md) to manage the the use of `beta` features. Conditional compilation is unnecessary, and not available, in the hero samples.
+
+[hero-chat]: https://github.com/Azure-Samples/communication-services-web-chat-hero
+[hero-calling]: https://github.com/Azure-Samples/communication-services-web-calling-hero
+[samples-chat]: ../../samples/Chat/
+[samples-calling]: ../../samples/Calling/
+
+
+## When to update the hero samples
+
+You should typically update hero samples for two distinct reasons:
+
+- To address any critical bugs in the hero sample.
+  - The most common example of this are security issues discovered by [depandabot](https://github.com/dependabot) filed as GitHub issues on the hero sample repositories.
+- To update the hero samples from the corresponding upstream samples.
+  - Hero samples showcase the latest [stable release of `@azure/communication-react`](https://www.npmjs.com/package/@azure/communication-react).
+  - You should update the hero samples from upstream every time we release a new stable version for `@azure/communication-react`, to showcase new features being shipped.
+
+
+## How To: Fix critical issues
+
+To fix any critical issues in the hero samples, including depandabot alerts in the hero sample repositories:
+
+- First, fix the issue in the upstream sample.
+- Then, port over just the fix to the hero sample.
+  - Porting over the fix is a manual process. In most cases, this simply involves updating the version of a package.json `dependencies` entry.
+  - The GitHub PR template for hero samples includes a section to add upstream PR references. Link to the upstream PR in this section.
+
+
+## How To: Update from upstream
+
+Updating hero samples from upstream samples is a manual process. You can minimize the toil by following the process recommended in this section.
+
+This section uses [an update to the chat hero sample](https://github.com/Azure-Samples/communication-services-web-chat-hero/pull/69) as an example. Follow similar steps when updating the other hero samples.
+
+- Pick the git commit in this monorepository to update from.
+  - By default, pick the git tag for the latest stable release. This ensures that the upstream sample is consistent with the released `@azure/communication-react` version.
+  - You may want to pick a later commit to pull in some useful updates to the sample.
```

Comment:
I would suggest to ignore this to keep instructions simple and only have choosing the git tag. We could add a section "out-of-band-hero-sample-updates" where app-only updates in between stable releases can be cherry picked in manually

Reply:
Agreed. Dropped those points.